cert-manager/namespaces/cert-manager/Deployment-cert-manager-cainjector.yaml

```diff
@@ -1,46 +1,39 @@
-# Source: cert-manager/templates/cainjector-deployment.yaml
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cert-manager-cainjector
-  namespace: "cert-manager"
   labels:
     app: cainjector
-    app.kubernetes.io/name: cainjector
+    app.kubernetes.io/component: cainjector
     app.kubernetes.io/instance: cert-manager
-    app.kubernetes.io/managed-by: Tiller
-    helm.sh/chart: cert-manager-v0.13.1
+    app.kubernetes.io/name: cainjector
+  name: cert-manager-cainjector
+  namespace: cert-manager
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: cainjector
-      app.kubernetes.io/name: cainjector
+      app.kubernetes.io/component: cainjector
       app.kubernetes.io/instance: cert-manager
-      app.kubernetes.io/managed-by: Tiller
+      app.kubernetes.io/name: cainjector
   template:
     metadata:
       labels:
         app: cainjector
-        app.kubernetes.io/name: cainjector
+        app.kubernetes.io/component: cainjector
         app.kubernetes.io/instance: cert-manager
-        app.kubernetes.io/managed-by: Tiller
-        helm.sh/chart: cert-manager-v0.13.1
-      annotations:
+        app.kubernetes.io/name: cainjector
     spec:
-      serviceAccountName: cert-manager-cainjector
       containers:
-        - name: cert-manager
-          image: "quay.io/jetstack/cert-manager-cainjector:v0.13.1"
-          imagePullPolicy: IfNotPresent
-          args:
-          - --v=2
-          - --leader-election-namespace=kube-system
-          env:
-          - name: POD_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          resources:
-            {}
-            
+      - args:
+        - --v=2
+        - --leader-election-namespace=kube-system
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/jetstack/cert-manager-cainjector:v1.1.0
```

Comment:
This is really a small nit, but replicating this to the local GCP repo is something we want to do in a future?

Reply:
I think this has come up before around the time quay.io had outages - given how widely used cert-manager is, if we were to expose a public mirror of cert-manager it could incur substantial cost (ref https://github.com/kubernetes/k8s.io/pull/912)